components/Blog/BlogHeader/index.tsx

```diff
@@ -18,12 +18,13 @@ const BlogHeader: FC<BlogHeaderProps> = ({ category }) => {
   const currentFile =
     siteConfig.rssFeeds.find(item => item.category === category)?.file ??
     'blog.xml';
+  const ariaLabel = t('layouts.blog.blogHeader.RSS');
 
   return (
     <header className={styles.blogHeader}>
       <h1>
         {t('layouts.blog.title')}
-        <Link href={`/feed/${currentFile}`}>
+        <Link href={`/feed/${currentFile}`} aria-label={ariaLabel}>
```

Comment:
```suggestion
        <Link href={`/feed/${currentFile}`} aria-label={ t('layouts.blog.blogHeader.RSS')}>
```

You can do that like for subtitle

Reply:
yeah, I know but  I did it like the referenced  link  
like [here](https://github.com/nodejs/nodejs.org/blob/004c16e039dbda73ab5048a607118ae27acca29f/components/Common/ThemeToggle/index.tsx#L21)